corehq/apps/cloudcare/templates/formplayer/dependencies.html

```diff
@@ -69,4 +69,10 @@
   <script src="{% static 'cloudcare/js/form_entry/formnav-all.js' %}"></script>
   <script src="{% static 'cloudcare/js/form_entry/entrycontrols_full.js' %}"></script>
   <script src="{% static 'cloudcare/js/form_entry/fullform-ui.js' %}"></script>
+  <script src="{% static '@mapbox/mapbox-gl-geocoder/dist/mapbox-gl-geocoder.min.js' %}"></script>
```

Comment:
@biyeun  just checking to see if this was the right way to do this, change commit here https://github.com/dimagi/commcare-hq/pull/28179/commits/fe996dcb4587e2de3b4321a54f00d2127466c672